src/v/coproc/pacemaker.cc

```diff
@@ -76,9 +76,11 @@ ss::future<> pacemaker::start() {
 
 ss::future<> pacemaker::reset() {
     _offs.timer.cancel();
+    ntp_context_cache ncc = std::move(_ntps);
```

Comment:
if you're expecting `_ntps` to be empty after this I think you should use `std::exchange(_ntps, {})`